src/dyndep.cc

```diff
@@ -65,10 +63,9 @@ bool DyndepLoader::LoadDyndeps(Node* node, DyndepFile* ddf,
   }
 
   // Reject extra outputs in dyndep file.
-  for (DyndepFile::const_iterator oe = ddf->begin(); oe != ddf->end();
-       ++oe) {
-    if (!oe->second.used_) {
-      Edge* const edge = oe->first;
+  for (auto& oe : *ddf) {
```

Comment:
Apart from using `const auto&`, can we also rename the variable for clarity here?
E.g. use `dyndep_output` instead of the cryptic `oe`?

Reply:
I renamed the variable.
I added `const` and double-checked all the other loops and applied it where possible. Thanks for the review!